app/src/main/kotlin/com/hedvig/app/authenticate/UserViewModel.kt

```diff
@@ -44,15 +42,7 @@ class UserViewModel(
 
   private val mutableViewState = MutableStateFlow(ViewState())
   val viewState: StateFlow<ViewState>
-    get() = mutableViewState
-
-  private val _events = Channel<Event>(Channel.UNLIMITED)
-  val events = _events.receiveAsFlow()
-
-  sealed class Event {
-    object Logout : Event()
-    data class Error(val message: String?) : Event()
-  }
```

Comment:
This event was a trampoline from
view interaction -> message to VM -> event to view -> view navigates somewhere
So just moved it in the activity itself directly.